### PR TITLE
NO-DED fixing issues from platform-docs

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description
Dear reviewers, I have included two different cases reported on the #platform-docs. That is why I mark each case with a comment mentioning you, so you are not surprised and sure what to check :)

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/messaging/customer-chat-api/index.mdx [hunk 1]
```diff
@@ -636,6 +636,7 @@ Returns the dynamic configuration of a given group. It provides data to call [**
 | `organization_id`       | `string`  | The ID of the organization for which the dynamic configuration is returned.                                                          |
 | `client_limit_exceeded` | `bool`    | Specifies if the limit of customers having a chat was exceeded.                                                                      |
 | `domain_allowed`        | `bool`    | Specifies if `url` is configured as a trusted domain (configurable in the Agent Application).                                        |
+| `online_group_ids`      | `array`   | It is returned in response with all the groups' numbers where agents accept chats. It is not visible when there is no such group.    |
```
Comment (kamilatokarska) [inline src/pages/messaging/customer-chat-api/index.mdx:639]: Hello @remleinpiotr, please see the changes connected with the #platform-docs feedback we got. Everything was described in the [Slack thread](https://livechat.slack.com/archives/C2FAU97K2/p1659104442122659). Below, you can find changes in the v3.4 and v3.5

File: src/pages/getting-started/app-guides/properties/index.mdx [hunk 2]
```diff
@@ -22,7 +22,7 @@ Before you can configure the Properties building block, you need to complete the
 
 - Have an account in the <Placeholder id="DEVELOPER_CONSOLE_URL" />.
 - Create an app for which you'll create properties. If you don't know how to create an app, [here's the tutorial](/getting-started/livechat-apps/#creating-livechat-apps).
-- Add the  `properties--configuration:rw` scope in the App Authorization building block. This scope allows for creating and editing properties.
+- Add the  `properties.configuration:rw` scope in the App Authorization building block. This scope allows for creating and editing properties.
```
Comment (kamilatokarska) [inline src/pages/getting-started/app-guides/properties/index.mdx:25]: Hello @bkubiak, I have included two cases I was taking care of from the #platform-docs feedback we got during the last few days. Everything connected with our case was described in the [Slack thread](https://livechat.slack.com/archives/C2XMTMQ5S/p1659953161127989). This is the only change connected with our case.

File: src/pages/messaging/customer-chat-api/index.mdx (hunk 1 above)
Comment (kamilatokarska) [reply src/pages/messaging/customer-chat-api/index.mdx:639]: Yes, it sounds better, @remleinpiotr WDYT?

Comment (kamilatokarska) [inline src/pages/messaging/customer-chat-api/index.mdx:639]:
```suggestion
| `online_group_ids`      | `array`   | It is returned in response with all the group IDs where agents accept chats. It is not visible when there is no such group.    |
```

File: src/pages/messaging/customer-chat-api/v3.3/index.mdx [hunk 3]
```diff
@@ -637,6 +637,7 @@ Returns the dynamic configuration of a given group. It provides data to call [**
 | `group_id`              | `number`  | The ID of the group for which the dynamic configuration is returned.                                                                 |
 | `client_limit_exceeded` | `bool`    | Specifies if the limit of customers having a chat was exceeded.                                                                      |
 | `domain_allowed`        | `bool`    | Specifies if `url` is configured as a trusted domain (configurable in the Agent Application).                                        |
+| `online_group_ids`      | `array`   | It is returned in response with all the groups' numbers where agents accept chats. It is not visible when there is no such group.    |
```
Comment (kamilatokarska) [inline src/pages/messaging/customer-chat-api/v3.3/index.mdx:640]:
```suggestion
| `online_group_ids`      | `array`   | It is returned in response with all the group IDs where agents accept chats. It is not visible when there is no such group.    |
```

File: src/pages/messaging/customer-chat-api/v3.5/index.mdx [hunk 4]
```diff
@@ -630,6 +630,7 @@ Returns the dynamic configuration of a given group. It provides data to call [**
 | `organization_id`       | `string`  | The ID of the organization for which the dynamic configuration is returned.                                                          |
 | `client_limit_exceeded` | `bool`    | Specifies if the limit of customers having a chat was exceeded.                                                                      |
 | `domain_allowed`        | `bool`    | Specifies if `url` is configured as a trusted domain (configurable in the Agent Application).                                        |
+| `online_group_ids`      | `array`   | It is returned in response with all the groups' numbers where agents accept chats. It is not visible when there is no such group.    |
```
Comment (kamilatokarska) [inline src/pages/messaging/customer-chat-api/v3.5/index.mdx:633]:
```suggestion
| `online_group_ids`      | `array`   | It is returned in response with all the group IDs where agents accept chats. It is not visible when there is no such group.    |
```

**Review by remleinpiotr — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Review by bkubiak — APPROVED** [review]

**Pull request merged**